RussianFilter/sections/general_extensions.txt

```diff
@@ -380,6 +380,8 @@ game2day.ru##body > .container { margin-top: 0!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/31394
 vgtimes.ru#$#.adfshift  { display: none!important; }
 vgtimes.ru#$#body { background-image: none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/31698
+zozhnik.ru#$#body { background: #000000!important; margin-top: -300px!important; }
```

Comment:
Black background is not good. When you disable red background you will have this element:
    background: url(/i/bg.png) #addadf;